action.yaml

```diff
@@ -22,32 +22,43 @@ inputs:
     description: "Copa version to use"
   output:
     description: "Output filename"
+    default: "output.json"
```

Comment:
are we okay to always produce an output?

Reply:
Fixed - this should have been removed. We also needed to change the order of arguments to take the format arg before the output arg since we are always providing the format flag. This would have broke cases where we didn't have the output.